projects/github.com/restic/restic/package.yml

```diff
@@ -0,0 +1,22 @@
+distributable:
+  url: https://github.com/restic/restic/releases/download/v{{ version }}/restic-{{version}}.tar.gz
+  strip-components: 1
+
+display-name: restic
+
+versions:
+  github: restic/restic
+
+provides:
+  - bin/restic
+
+build:
+  script: |
+    go run build.go
+    mkdir -p "{{ prefix }}"/bin
+    mv restic "{{ prefix }}"/bin
+  dependencies:
+    go.dev: ^1.20
```

Comment:
The latest restic release is already built using Go 1.21.

Reply:
generally, this should be the _oldest_ release that will correctly build the binary (since ^1.x == >=1.x && <2)

Reply:
Ah, ok, then 1.20 is fine.